index.html

```diff
@@ -175,13 +175,52 @@ <h1>Skills</h1>
                     <li><span>&bull; GitLab</span></li>
                     <li><span>&bull; Terminal</span></li>
                 </ul>
-                <img class="about-img" src="assert/ic_languages_mob.svg" alt="lightbulb icon" />
+                <img class="about-img" src="/assert/ic_languages_mob.svg" alt="lightbulb icon" />
             </div>
+        </div>
+    </section>
+    <footer>
+        <div class="form-section">
+            <section class="section">
+                <h1 class="section-h"> Get started</h1>
+                <p class="section-p"> Lorem ipsum dolor sit amet, consectetur
+                    adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore et dolore magna.</p>
+            </section>
+            <form action="https://formspree.io/f/xknyprke" method="POST">
+                <ul>
 
+                    <li> <input class=" name-input" name="text" type="text" placeholder="Name">
+                    </li>
+                    <li> <input class="mail-input" name="email" type="email" placeholder="EMAIL"> </li>
+                    <li><textarea class="textarea-input" name="textarea" id="textarea" cols="30"
+                            rows="10"> hello </textarea></li>
```

Comment:
- [ ] Please use the same design as in the Figma design 

| Your design | Figma |
| ------------- | ------- |
| ![image](https://user-images.githubusercontent.com/74744787/165407233-9f7cee40-e19f-424a-a151-c14fae3333d1.png) | ![image](https://user-images.githubusercontent.com/74744787/165407274-a159ed97-daad-4550-8cbf-e171b8a704f0.png) |
| 1-notice how you don't have the same image in the corner as in Figma design,2- please make sure that the placeholder text is the same as in Figma design | |